pages/linux/ceph.md

```diff
@@ -0,0 +1,32 @@
+# ceph
+
+> Ceph implements object storage on a single distributed computer cluster, and provides interfaces for object, block and file-level storage.
+> More information: <https://ceph.io>.
+
+- Check or watch cluster health:
+
+`ceph {{status | -w}}`
```

Comment:
What's the difference between `status` and `-w` here? If they do the same thing, I think I'd prefer `status`, since it's more clear.

Reply:
Actually They both doing something with a little different. The `ceph -w` run in watch mode but `ceph status` write once the status. for more clarity, I'll remove the `-w`.